embc-app/ViewModels/Search/EvacueeSearchQueryParameters.cs

```diff
@@ -0,0 +1,32 @@
+using Gov.Jag.Embc.Public.Utils;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Gov.Jag.Embc.Public.ViewModels.Search
+{
+    public class EvacueeSearchQueryParameters : SearchQueryParameters
+    {
+        [FromQuery(Name = "last_name")]
+        public string LastName { get; set; }
+
+        [FromQuery(Name = "last_name")]
+        public string FirstName { get; set; }
+
```

Comment:
Please fix typo: this property should map to **first_name**

Reply:
Probably a good idea! :)